zbus/src/connection/mod.rs

```diff
@@ -526,12 +526,10 @@ impl Connection {
         W: TryInto<WellKnownName<'w>>,
         W::Error: Into<Error>,
     {
-        self.request_name_with_flags(
-            well_known_name,
-            RequestNameFlags::ReplaceExisting | RequestNameFlags::DoNotQueue,
-        )
-        .await
-        .map(|_| ())
+        let flags = BitFlags::default();
+        self.request_name_with_flags(well_known_name, flags)
```

Comment:
future suggestion: best to keep the style of the code intact: there was no need for a new variable.